### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by CryptoRafe**
Update all 3 README.md files of Wasabi to give viewers more information about the differences of these files and repositories.

Moved some of the info between files and added few comments and hyperlinks to all README.md files. 

Basic idea is this: 
WasabiDocs/readme = introduction to the Docs repo
WasabiDocs/docs/readme = manual and links to specific documents
WalletWasabi/readme = introduction to developers repo.

Haven't made a new PR to WalletWasabi yet.

PS. I'm new to github so bare with me guys.

### CODE REVIEW

**Comment by CryptoRafe** [comment]
Moved some of the info to WalletWasabi README.md file. I think some more info is needed there. Deleting some of the same stuff from here is not necessarily needed

**Comment by MaxHillebrand** [comment]
So far, concept NACK.

We have many different repos, and three different readme's, so let me try to explain which one is for what:

- `wasabidoc/docs/readme.md` is the first impression on the docs page, and this should be tailor made for future-coiners and Wasabi n00bs, with very basic explanations about Wasabi.
- `wasabidoc/reamde.md` is not visible on the docs webpage, and thus the first impression for those who contribute to the docs itself, and with all the info they will need.
- `walletwasabi/readme.md` is only visible in the github repo with all the code, and thus it should be made to fit the required info that code contributors need. 

Thanks @CryptoRafe for getting started with this!
But could you please re-work this PR so that it moves us towards the goal of having these three readme's well explain the different situations?
You can keep the PR open and commit further changes to your branch `patch-1`. I'll review again when you are ready :kissing_heart:

**Comment by MaxHillebrand** [comment]
Is it "**from** the docs file" or "**in** the docs file"?

File: README.md [hunk 1]
```diff
@@ -1,5 +1,14 @@
...
+This is the open source documentation repository of [Wasabi Wallet](https://wasabiwallet.io)*. Developers repository can be found [here](https://github.com/zkSNACKs/WalletWasabi).
+
+From the [docs](https://github.com/zkSNACKs/WasabiDoc/tree/master/docs) files you will find information about the nuances of privacy in Bitcoin, how Wasabi solves these difficult problems, and how you can use these tools to defend yourself. 
+If you would like to support the project by educating your peers, this documentation repository is the right place for your efforts!
+
+Have a question that is not yet covered? Please [open an issue](https://github.com/zkSNACKs/WasabiDoc/issues).
+If you have a good answer to a question, please [open a pull request](https://github.com/zkSNACKs/WasabiDoc/pulls).
+For more details on how to contribute, see the [contribution checklist](/docs/ContributionChecklist.md) or [To-Do-List](https://github.com/zkSNACKs/WasabiDoc/blob/master/docs/ToDo.md).
+
+*[Wasabi Wallet](https://wasabiwallet.io) is an open-source, non-custodial, privacy focused Bitcoin wallet for desktop.
```
Comment (MaxHillebrand) [inline README.md]: Why not make this line the second line, after "This is the open source documentation repository...", and remove the star?  Seems to be easier to read without jumping across the text.

**Comment by MaxHillebrand** [comment]
FYI, this got re-worked too the new repo structure and merged in #35

**Pull request closed without merging**